docs/advanced-features/open-telemetry.md

```diff
@@ -94,87 +94,127 @@ In order to be able to leverage instrumentation provided by next.js you will nee
 
 ### `fetch [http.method] [http.url]`
 
+- `next.span_type`: `AppRender.fetch`
+
 This span represents the fetch request executed in your code.
 
 Attributes:
 
-- `http.method`: The HTTP method of the fetch request (e.g., GET).
-- `http.url`: The URL of the fetch request (e.g., `https://vercel.com/`).
-- `net.peer.name`: The domain name of the fetch request (e.g., `vercel.com`).
-- `next.span_type`: `AppRender.fetch`
+- [Common HTTP attributes](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/http/#common-attributes)
+  - `http.method`
+- [Client HTTP attributes](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/http/#http-client)
+  - `http.url`
+  - `net.peer.name`
+  - `net.peer.port` (only if specified)
+- `next`
+  - `next.span_name`
+  - `next.span_type`
 
 ### `executing api route (app) [next.route]`
 
-This span represents the execution of an API route handler in the application.
+- `next.span_type`: `AppRouteRouteHandlers.runHandler`.
+
+This span represents the execution of an API route handler in app router.
 
 Attributes:
 
-- `next.route`: The route pattern of the request (e.g., `/[param]/getUser`).
-- `next.span_type`: `AppRouteRouteHandlers.runHandler`.
+- `next`
+  - `next.span_name`
+  - `next.span_type`
+  - `next.route`
 
 ### `getServerSideProps [next.route]`
 
+- `next.span_type`: `Render.getServerSideProps`.
+
 This span represents the execution of `getServerSideProps` for a specific route.
 
 Attributes:
 
-- `next.route`: The route pattern of the request (e.g., `/[param]/user`).
-- `next.span_type`: `Render.getServerSideProps`.
+- `next`
+  - `next.span_name`
+  - `next.span_type`
+  - `next.route`
 
 ### `getStaticProps [next.route]`
 
+- `next.span_type`: `Render.getStaticProps`.
+
 This span represents the execution of `getStaticProps` for a specific route.
 
 Attributes:
 
-- `next.route`: The route pattern of the request (e.g., `/[param]/user`).
-- `next.span_type`: `Render.getStaticProps`.
+- `next`
+  - `next.span_name`
+  - `next.span_type`
+  - `next.route`
 
 ### `render route (pages) [next.route]`
 
+- `next.span_type`: `Render.renderDocument`.
+
 This span represents the process of rendering the document for a specific route.
 
 Attributes:
 
-- `next.route`: The route pattern of the request (e.g., `/[param]/user`).
-- `next.span_type`: `Render.renderDocument`.
+- `next`
+  - `next.span_name`
+  - `next.span_type`
+  - `next.route`
 
 ### `generateMetadata [next.page]`
 
-This span represents the process of generating metadata for a specific route.
+- `next.span_type`: `ResolveMetadata.generateMetadata`.
+
+This span represents the process of generating metadata for a specific page (single route can have multiple of these spans).
 
 Attributes:
 
-- `next.page`: The page pattern for which the metadata is generated (e.g., `/app/[param]/layout` or `/app/[param]/page`).
-- `next.span_type`: `ResolveMetadata.generateMetadata`.
+- `next`
```

Comment:
I don't think we need those to be indented

Reply:
Removed `next` group from all spans.